setup.py

```diff
@@ -110,11 +110,11 @@ def _format_pulp_requirement(plugin, specifier=None, ref=None, gh_namespace="pul
 
 requirements = [
     "galaxy-importer==0.4.5",
-    _format_pulp_requirement("pulpcore", ref="6e44fb2fe609f92dc1f502b19c67abd08879148f"),
```

Comment:
@awcrosby if `_format_pulp_requirement` doesn't work, can you add a comment with an example for pinning pulp  to a git commit and some notes about how to pin to a commit?

Reply:
@newswangerd I don't know how to pin pulp deps to a commit and get it to work with both pip-tools and pulp ci. I feel like the current `_format_pulp_requirement` is going in the right direction. I can put a fixme on that method noting its limitations, how does that sound?

Reply:
alright, that sounds good